Chemistry/tests/SamplesTests/DocsInvokingChemistryLibrary.cs

```diff
@@ -0,0 +1,153 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// This test ensures that any chemistry library syntax changes
+// that affect samples are detected.
+
+#region Using Statements
+// We will need several different libraries in this sample.
+// Here, we expose these libraries to our program using the
+// C# "using" statement, similar to the Q# "open" statement.
+
+// We will use the data model implemented by the Quantum Development Kit Chemistry
+// Libraries. This model defines what a fermionic Hamiltonian is, and how to
+// represent Hamiltonians on disk.
+using Microsoft.Quantum.Chemistry.OrbitalIntegrals;
+using Microsoft.Quantum.Chemistry.Fermion;
+using Microsoft.Quantum.Chemistry.QSharpFormat;
+
+// The System namespace provides a number of useful built-in
+// types and methods that we'll use throughout this sample.
+using System;
+
+// We use this for convnience functions for manipulation arrays.
+using System.Linq;
+
+//
+using Xunit;
+#endregion
+
+namespace Microsoft.Quantum.Chemistry.Tests.Docs
+{
+    public static class InvokingChemistryLibrary
+    {
+        [Fact]
+        static void MakeHamiltonian()
+        {
+            // These orbital integrals are represented using the OrbitalIntegral
+            // data structure.
+            var energyOffset = 0.713776188;
+            var nElectrons = 2;
+            var orbitalIntegrals = new OrbitalIntegral[]
+            {
+                new OrbitalIntegral(new[] { 0,0 }, -1.252477495),
+                new OrbitalIntegral(new[] { 1,1 }, -0.475934275),
+                new OrbitalIntegral(new[] { 0,0,0,0 }, 0.674493166),
+                new OrbitalIntegral(new[] { 0,1,0,1 }, 0.181287518),
+                new OrbitalIntegral(new[] { 0,1,1,0 }, 0.663472101),
+                new OrbitalIntegral(new[] { 1,1,1,1 }, 0.697398010),
+                // Add the identity term
+                new OrbitalIntegral(new int[] { }, energyOffset)
+            };
+
+            // We initialize a fermion Hamiltonian data structure and add terms to it
+            var fermionHamiltonian = new OrbitalIntegralHamiltonian(orbitalIntegrals).ToFermionHamiltonian();
+
+            // The Jordan-Wigner encoding converts the Fermion Hamiltonian, 
```

Comment:
We've used a lowercase "f" for fermion (this seems to be in keeping with [historical usage of the term](https://books.google.com/ngrams/graph?content=fermion%2CFermion&year_start=1950&year_end=2000&corpus=15&smoothing=3&share=&direct_url=t1%3B%2Cfermion%3B%2Cc0%3B.t1%3B%2CFermion%3B%2Cc0#t1%3B%2Cfermion%3B%2Cc0%3B.t1%3B%2CFermion%3B%2Cc0)).
```suggestion
            // The Jordan–Wigner encoding converts the fermion Hamiltonian, 
```

Reply:
(As a side note, "Hamiltonian" is still strongly preferred to "hamiltonian" in historical usage.)